### DISCUSSION TITLE
services.AddMediatR now gives me error in Blazor WASM

### DISCUSSION THREAD

**Discussion by rquirosIseco** [LuckyPennySoftware/MediatR#849] [Q&A]
Hey guys, sorry if it's a dumb question but with version 11.00 i had this code line:

` services.AddMediatR(Assembly.GetExecutingAssembly());`

And now with v12.00 gives me the next error:

![image](https://user-images.githubusercontent.com/116882155/219654781-c7f6861b-66cb-44bf-9cc9-f197e0fd1f2b.png)

How can i solve this?

Thanks.

**Comment by jbogard** [comment]
Check the [migration guide](https://github.com/jbogard/MediatR/wiki/Migration-Guide-11.x-to-12.0). There's a section at the bottom.
